lib/cext/preprocess.rb

```diff
@@ -55,6 +55,8 @@ def preprocess(line)
 end
 
 if __FILE__ == $0
+  puts "# 1 \"#{ARGF.filename}\""
```

Comment:
Where did you see this syntax? It might actually be something a specific preprocessor recognizes but not another.